client/rpc-api/src/chain/mod.rs

```diff
@@ -54,6 +55,23 @@ pub trait ChainApi<Number, Hash, Header, SignedBlock> {
 	#[rpc(name = "chain_getFinalizedHead", alias("chain_getFinalisedHead"))]
 	fn finalized_head(&self) -> Result<Hash>;
 
+	/// Set offchain storage under given key and prefix.
+	#[rpc(name = "chain_setOffchainStorage")]
+	fn set_offchain_storage(
+		&self,
+		prefix: StorageKey,
```

Comment:
Could we rather keep the enum instead of allowing any kind of prefix directly? I think this will get sorted as well if you switch to use `OffchainWorkers` structure directly instead of going through `Client` to low-level `OffchainStorage` db.

Reply:
You mean prefix must be StorageKind type, right?